src/android/AudioPlayer.java

```diff
@@ -346,8 +352,12 @@ public void resumeRecording() {
      * Start or resume playing audio file.
      *
      * @param file              The name of the audio file.
+     * @param output            Type of output device: speaker/earpiece/no change
      */
-    public void startPlaying(String file) {
+    public void startPlaying(String file, String output) {
+        if (!output.equals(OUTPUT_NO_CHANGE)) {
```

Comment:
defend against null exception in output
```suggestion
        if (!output.equals(OUTPUT_NO_CHANGE)) {
```

Reply:
done